test/unit/presenters/formats/popular_links_format_presenter_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class PopularLinksFormatPresenterTest < ActiveSupport::TestCase
+  def subject
+    Formats::PopularLinksPresenter.new(FactoryBot.create(:popular_links))
+  end
+
+  context "#render_for_publishing_api" do
+    setup do
+      @result = subject.render_for_publishing_api
+    end
+
+    should "have expected required fields" do
+      assert_equal "Homepage Popular Links", @result[:title]
+      assert_equal "link_collection", @result[:schema_name]
+      assert_equal "link_collection", @result[:document_type]
+      assert_equal "publisher", @result[:publishing_app]
+      assert_equal "frontend", @result[:rendering_app]
+      assert_equal ({ link_items: [{ url: "https://www.url1.com", title: "title1" },
+                                   { url: "https://www.url2.com", title: "title2" },
+                                   { url: "https://www.url3.com", title: "title3" },
+                                   { url: "https://www.url4.com", title: "title4" },
+                                   { url: "https://www.url5.com", title: "title5" },
+                                   { url: "https://www.url6.com", title: "title6" }] }),
+                   @result[:details]
+    end
+
+    should "have expected optional fields" do
+      assert_not_empty @result[:access_limited]
```

Comment:
Maybe we can test the actual `access_limited` value, [like how the edition format presenter test does](https://github.com/alphagov/publisher/blob/cd76211dda73c9bb2dbc96d3c5bb126413cd6f9d/test/unit/presenters/formats/edition_format_presenter_test.rb#L159).